stage #1/module #1. Exceptions/content.md

```diff
@@ -1,20 +1,175 @@
-# How to write your own exception classes?
+# Exceptions
+
 
 ## Lesson Agenda
++ What is an Exception?
++ The Three Kinds of Exceptions
++ How to Catch and Handle Exceptions?
++ The Exceptions Thrown by a Method
++ How to write your own exception classes?
++ Materials
 
-+ Intro
-+ Why do I need custom exceptions?
-+ Writing your own exception class
-+ Re-throwing an exception which is wrapped in a custom exception
-+ Best practices for custom exceptions
-+ Custom Exception syntax
 
-### Intro
+## What is an Exception?
+
+### Throwing an exception
+**An exception** is an event that occurs during the execution of a program that disrupts the normal flow of instructions.
+
+**Throwing an exception** is the process of creating an exception object and handing it to the runtime system.
+
+![image](./media/image1.png)
```

Comment:
Could you use names that will reflect the information of the file?

Reply:
Sure, thanks